tso-server/src/main/java/com/yahoo/omid/tso/RetryProcessorImpl.java

```diff
@@ -99,24 +109,30 @@ public void onEvent(final RetryEvent event, final long sequence, final boolean e
 
     }
 
-    private void handleCommitRetry(RetryEvent event) throws InterruptedException, ExecutionException {
+    public void addAbort(Batch batch, long startTimestamp, Channel c, MonitoringContext context) {
```

Comment:
These functions are not needed. I used these in some intermediate version. 
Removed.